build-linux.sh

```diff
@@ -3,11 +3,24 @@ set -e
 
 export PATH="$HOME/miniconda/bin:$PATH"
 
+# add flat buffers
+cd ..
+git clone https://github.com/google/flatbuffers.git
+cd ./flatbuffers
+git checkout 925c1d77fcc72636924c3c13428a34180c30f96f  # release commit
```

Comment:
Can you change the comment to which release of flatbuffers we are targeting?

Reply:
added.